.drone.star

```diff
@@ -2196,6 +2196,7 @@ def ocisService():
                 "STORAGE_USERS_DRIVER_OWNCLOUD_DATADIR": "/srv/app/tmp/ocis/owncloud/data",
                 "WEB_ASSET_PATH": "%s/dist" % dir["web"],
                 "WEB_UI_CONFIG": "/srv/config/drone/config-ocis.json",
+                "FRONTEND_ENABLE_RESHARING": "true",
```

Comment:
Resharing in oCIS is currently opt-in?

Reply:
yes, optin

Reply:
at least for now, we've decided that we have it around in ocis but do not enable it by default to not put beta in danger